python/ray/runtime_env/runtime_env.py

```diff
@@ -349,16 +355,19 @@ def __init__(
         if not _validate:
             return
 
-        if self.get("conda") and self.get("pip"):
+        use_conda_install = int(self.get("conda") is not None)
+        use_pip_install = int(self.get("pip") is not None)
+        use_uv_install = int(self.get("uv") is not None)
+        if use_conda_install + use_pip_install + use_uv_install > 1:
             raise ValueError(
-                "The 'pip' field and 'conda' field of "
-                "runtime_env cannot both be specified.\n"
-                f"specified pip field: {self['pip']}\n"
-                f"specified conda field: {self['conda']}\n"
-                "To use pip with conda, please only set the 'conda' "
-                "field, and specify your pip dependencies "
-                "within the conda YAML config dict: see "
-                "https://conda.io/projects/conda/en/latest/"
+                "The 'pip' field, 'uv' field, and 'conda' field of "
+                "runtime_env cannot be specified at the same time.\n"
+                f"specified pip field: {self.get('pip')}\n"
+                f"specified conda field: {self.get('conda')}\n"
+                f"specified conda field: {self.get('uv')}\n"
+                "To use pip with conda or uv, please only set the 'conda' or 'uv'"
+                "field, and specify your pip dependencies within the conda YAML "
+                "config dict: see https://conda.io/projects/conda/en/latest/"
```

Comment:
```suggestion
                "To use pip with conda, please only set the 'conda'"
                "field, and specify your pip dependencies within the conda YAML "
                "config dict: see https://conda.io/projects/conda/en/latest/"
```

You can use pip inside conda but not inside uv.

Reply:
I see what you mean, updated.